### PR TITLE
Add florent zara permission in order to build ort project

### PR DESCRIPTION
**Pull request by heurtematte**
Related to https://gitlab.eclipse.org/eclipsefdn/infrazilla/-/issues/1033

### CODE REVIEW

**Comment by mbarbero** [comment]
You don't need to do that at the jenkins level. Please grant him access to the `ORT` folder only

**Comment by mbarbero** [comment]
> You don't need to do that at the jenkins level. Please grant him access to the `ORT` folder only

too fast, I re-check and you give him the bare minimal access, got it 🤦

**Comment by fredg02** [comment]
yeah, he needs the "Agent/Build" permission that can't be given on a folder level. :unamused:

File: instances/foundation-internal.infra/target/k8s/known-hosts.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
...
...gitlab.eclipse.org,141.94.179.217 ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBIPik0+gkBsbyVzTLvDq4ESZJMlUSZeb/tjdOHPbTLcSSlHX8R4CdqQwjWdxi4AMQ/76FXUsNH2azvkvgSh/r+E=\n[foundation.eclipse.org]:29418,[172.25.25.206]:29418 ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEArAdc5w86HX4vPtqpUVir4WuNWJVTLv8f5X3UuCxbCzlYUljvRRteW4O19D2Ud4ClHK+Bw54nYW1kKgPslUGcM6Qeb3iPLK43TQOqjOYDSjy11iXeZ0SaYLsGgibRPT+VLZGAHqbmXvYxuK68vuCQCwgAQ1e02OcAQMX0t/0TboaFtdz0dQeX6y3gztpw87HzNOzKBxbJ13cMlqqysXoZ420huu8+oSwGfIHCRl8z39KnNmBxyOpE/Kzwl0YvGIZSPVGOuuI+24R53bNYT69n4cOuAk/VTa7JAC+i9cN6T3tDDDk9TrKezZtKvHYLZDXEIOCgj56rB7kvA+Ub0jnAgw==\n20.104.233.12 ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQDkH1ET4ScUfV5JHHr7f7dOg4jHg09SL2Q03ST6oAmC5yxmU52uu8vhUZRM5H5zjrW4ml7/U4s+J3LyUHNtcQq5xqBrQCDc2uSXUTFeShp3FeehHRvvOzlJqSANvQj+0Qdxb/aU0lxAYXaLsRy+HEH75aSceTXYDFvVjygteBApVFAaTCNYlDNPsxMUQ1ZKi7cy7DPT1w6PVpGO+9i7WXHkMCJlhOLf6ivJWgH5NtcHJMAxs7fGHYPjn8rUJlSAY/Hg1bT5JcGAdNWxsaCmKMucaGBeHz2E+hXtx6k4ye5fJqlFuWkxfWyJSewmoNrilACl9XKp5mPbVZyCZCDJCtVYkT6gf3nm5bTiSfIBb5fLM1eVKssa2PNcEW7sajRKEpw6GKcD0a1gj1ERbZapVWk42AD9TCIeaT8SniZrq60n5E/SVRBQvoCArdLIQgfm4zflUsH/gFeMFMV/PvQ6vrbRCd+fhbnaRWFVOpUUwwMaM+8OXCbS9XOPge8B2wqOVqk=\n"
```
Comment (pstankie) [inline instances/foundation-internal.infra/target/k8s/known-hosts.json:4]: We don't need 
`gitlab.eclipse.org ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIBsuD5uVWgjZX3jrcvdWu0DwbbarL32mfbNAVABi4wCc` ?

**Pull request merged**